default.nix

```diff
@@ -14,9 +14,5 @@ in naersk.buildPackage {
       pkgs.perl
   ];
 gitSubmodules = true;
-  shellHook =
-  ''
-    SRC_DIR="$( cd -- "$( dirname -- "''${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
-    cp -r $out/target $SRC_DIR/target
-  '';
```

Comment:
I'm very confused what this did in the first place :laughing:

Reply:
It was initially used for integration testing but now we let `nix-shell` handle it